django/db/backends/postgresql/base.py

```diff
@@ -149,6 +150,11 @@ class DatabaseWrapper(BaseDatabaseWrapper):
     # PostgreSQL backend-specific attributes.
     _named_cursor_idx = 0
 
+    def get_database_version(self):
+        # Return a tuple of database version.
```

Comment:
"of the" (though maybe it's redundant with the docstring if we add one to the base class)... in that case, the "E.g." still seems helpful.